_includes/_head.html

```diff
@@ -7,84 +7,71 @@
 #   Variables › {{ description }} › excerpt, page.meta_description, or site.description
 #   Variables › {{ title }} › page.title
 #
-{% endcomment %}{% capture ignore %}
+{% endcomment %}
 
 {% assign url = site.baseurl | prepend: site.url %}
 
-{% capture description %}{% if page.meta_description %}{{ page.meta_description }}{% elsif page.teaser %}{{ page.teaser }}{% elsif page.excerpt %}{{ page.excerpt }}{% elsif site.description %}{{ site.description }}{% endif %}{% endcapture %}
-{% assign description = description | strip_html | escape | strip %}
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-{% capture title %}{% if page.meta_title %}{{ page.meta_title}}{% elsif page.title %}{{ page.title }}{% else %}{{ site.title | strip_html | strip }}{% endif %}{% endcapture %}
+<link rel="stylesheet" type="text/css" href="{{ url }}/assets/css/styles_feeling_responsive.css">
 
-{% assign title = title | strip_html | escape | strip_newlines %}
+{% if site.asciidoctor-enabled == true %}
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.0/css/font-awesome.min.css">
+{% endif %}
 
-{% endcapture %}<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>{{ title }}</title>
+<script src="{{ url }}/assets/js/modernizr.min.js"></script>
 
-    <link rel="stylesheet" type="text/css" href="{{ url }}/assets/css/styles_feeling_responsive.css">
+<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
+<script>
+WebFont.load({
+google: {
+families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
+}
+});
+</script>
```

Comment:
This stuff wasn't actually added. I converted tabs to spaces in this file and, well, it seems like an `.editorconfig` may be in order. Thoughts?